std/experimental/checkedint.d

```diff
@@ -980,6 +985,17 @@ if (isIntegral!T || is(T == Checked!(U, H), U, H))
         x += 1;
         assert(MyHook.thereWereErrors);
     }
+
+    @system unittest
+    {
+        auto x1 = Checked!int(10);
+        auto x2 = Checked!int(10);
+        x1 += x2;
+        assert(x1.get == 20);
+        auto x3 = Checked!(Checked!int)(10);
+        x1 += x3;
+        assert(x1.get == 30);
```

Comment:
Would be great to add two more tests:
- different hook
- different type, but one that coerces to int (e.g. `short`)